---
title: Projects
date: 2022-10-24

type: landing

sections:
  - block: markdown
    content:
      title: Projects
      subtitle: 
      text: |
        |   |  |
        |--------------------|---|
        | <h3>CancerMine</h3>This project uses natural language processing to identify cancer genes and their roles in different cancers (e.g. as drivers, oncogenes or tumor suppressors). This information can be used to help identify important cancer mutations and understanding the underlying genetics of different cancer types. This work was published in <a href="https://doi.org/10.1038/s41592-019-0422-y">Nature Methods</a>. The dataset can be explored through <a href="http://bionlp.bcgsc.ca/cancermine/">this web viewer</a>, the code is available at <a href="https://github.com/jakelever/cancermine">GitHub</a> and the dataset is available at <a href="https://doi.org/10.5281/zenodo.1156241">Zenodo</a> | ![icon for the cancer mine project](cancermine.jpg) |
        | <h3>CIViCmine</h3>Precision oncology enables scientists and clinicians to probe the genetics of individual patients' tumours. But the clinical relevance of each mutation can be hard to ascertain and reference to the latest research is often required. The CIViC database aims to curate this expert knowledge. As part of this, the CIViCmine project uses natural language processing to identify mentions of cancer mutations and their clinical impacts. This work was published in <a href="https://doi.org/10.1186/s13073-019-0686-y">Genome Medicine</a>. The dataset can be explored through <a href="http://bionlp.bcgsc.ca/civicmine/">this web viewer</a>, the code is available at <a href="https://github.com/jakelever/civicmine">GitHub</a> and the dataset is available at <a href="https://doi.org/10.5281/zenodo.1472826">Zenodo</a>. | ![icon for the civicmine project](civicmine.jpg) |
        | <h3>CoronaCentral</h3>During the coronavirus pandemic, a vast number of research papers were published related to COVID-19. The Corona Central resource categorized these using a BERT-based classifier along with a unique dataset of categorized documents. It provided a portal to explore these research papers. This work was published in <a href="https://doi.org/10.1073/pnas.2100766118">PNAS</a>. The code can be accessed at <a href="https://github.com/jakelever/corona-ml">GitHub</a> and the corpus of documents at <a href="https://doi.org/10.5281/zenodo.4383289">Zenodo</a>. | ![icon for the corona central project](coronacentral.jpg) |
        | <h3>FusionDTI</h3>Predicting drug-target interaction (DTI) is critical in the drug discovery process. Despite remarkable advances in recent DTI models through the integration of representations from diverse drug and target encoders, such models often struggle to capture the fine-grained interactions between drugs and protein. To address this issue, we introduce a novel model, called FusionDTI, which uses a token-level Fusion module to effectively learn fine-grained information for Drug-Target Interaction. This work was published in <a href="https://zhaohanm.github.io/FusionDTI.github.io/">ICML 2024 AI for Science Workshop</a>. The code can be accessed at <a href="https://github.com/zhaohanm/fusiondti">GitHub</a> and the demo at <a href="https://huggingface.co/spaces/Gla-AI4BioMed-Lab/FusionDTI">Hugging Face</a>. | ![icon for the FusionDTI project](FusionDTI.png) |
        | <h3>FusionGDA</h3>Identifying associations between genes and diseases is critical for diagnosis, prevention, prognosis and drug development. We propose a novel FusionGDA model, which utilises a pre-training phase with a fusion module to enrich the gene and disease semantic representations encoded by pre-trained language models. This work was published in <a href="https://academic.oup.com/bib/article/25/5/bbae380/7735275">Briefings in Bioinformatics</a>. The code can be accessed at <a href="https://github.com/ZhaohanM/FusionGDA">GitHub</a> and the demo at <a href="https://huggingface.co/spaces/Gla-AI4BioMed-Lab/FusionGDA">Hugging Face</a>. | ![icon for the FusionGDA project](FusionGDA.jpg)|
        | <h3>BPP</h3>Biological pathways are a series of interconnected biochemical reactions that support life activities. Current research relies heavily on experiments and manual analysis, overlooking the rich topological information within pathway networks. We develop a Biochemical Pathway Prediction (BPP) platform to automatically identify potential connections within these pathways. BPP can predict participants or products in biochemical reactions and includes tools to interpret these predictions. This work was published in <a href="https://academic.oup.com/bib/article/25/5/bbae355/7724465">Briefings in Bioinformatics</a>. The website and code can be accessed at <a href="https://github.com/Glasgow-AI4BioMed/BPP">GitHub</a>. | ![icon for the BPP project](BPP.jpg)|

---
